bot/seasons/evergreen/bookmark.py

```diff
@@ -39,27 +52,80 @@ def __init__(self, bot: commands.Bot):
         embed = discord.Embed(
             title=title,
```

Comment:
setting `title` as the title of the embed will break discord's special format, such as when bookmarking using a channel as the name, like this

![image](https://user-images.githubusercontent.com/19921765/78245409-a3ba2280-7511-11ea-9bb4-df9408c3f16f.png)
![image](https://user-images.githubusercontent.com/19921765/78245421-a9b00380-7511-11ea-907f-21cef944ad4c.png)

You may want to put it in the description instead.

Reply:
You want me to break the symmetry how it used to work previously
if yes What you think title will be so that it is easy to find ?

Reply:
Actually, you can definitely use `.clean_content` of the message, that should escape and show the name of the channel properly.

If you need to escape markdown as well, you will need to look at [discord.utils.escape_markdown](https://discordpy.readthedocs.io/en/latest/api.html#discord.utils.escape_markdown)

Reply:
I think you mean
https://discordpy.readthedocs.io/en/latest/api.html#discord.utils.escape_mentions
But that does not contain channel mention.
Guess we can't make everything work.

Reply:
It is linked in the comment above.

And as I've said, you can certainly use `.clean_content` to escape mentions for user / channels.

Reply:
clean_content can only be applied on message object not string I can do that on target.clean_content not title.

Reply:
I think you can, you are forgetting `ctx.message` in this case. Either way, the format on the title should be addressed, as `<#267659945086812160>` does not make a great title.

Reply:
ctx.message.content will return whole command string including `.bm @user1`

Reply:
I can trick discord.py actually but i don't feel like that is a good way to do that 
basically change ctx.message.content = title
that way I can use clean_content

Reply:
![image](https://user-images.githubusercontent.com/45149585/78338589-af105b00-75b0-11ea-9ddc-0994898ead77.png)
![image](https://user-images.githubusercontent.com/45149585/78338601-b33c7880-75b0-11ea-8a79-3ecbf24e8423.png)
I mean why not ?

Reply:
Maybe should better remove title as `title` and use:
```
**Title:** {title}
{content}
```
as description?

Reply:
Ohh this got a good story
If a message is exactly discord message length
this will crash the command as now description is higher than limit

Reply:
Hmm. Yes. For this you should make length check, like 1900 characters (2000 is Discord max).

Reply:
#337 you should read this ,
And i don't think there is need to change how people are familiar with bookmark, for example if you see all the bm at once and they are different it can be hard to understand,

Reply:
![image](https://user-images.githubusercontent.com/45149585/78340311-7d4cc380-75b3-11ea-83b4-2489785a2de6.png)
![image](https://user-images.githubusercontent.com/45149585/78340340-8d64a300-75b3-11ea-9ee2-693739504dbb.png)

This is perfect.